qa/tasks/mgr/dashboard/test_mgr_module.py

```diff
@@ -4,7 +4,7 @@
 import logging
 import requests
 
-from .helper import DashboardTestCase, JAny, JObj, JList, JLeaf
```

Comment:
I think, this should already work in py3

Reply:
if i try to import this file in python3, i have
```
Importing module test_mgr_module from tasks/mgr/dashboard/test_mgr_module.py
Traceback (most recent call last):
  File "test_import.py", line 37, in <module>
    _import_file(p)
  File "test_import.py", line 19, in _import_file
    mod_spec.loader.exec_module(mod)
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "tasks/mgr/dashboard/test_mgr_module.py", line 7, in <module>
    from .helper import DashboardTestCase, JAny, JObj, JList, JLeaf
ImportError: attempted relative import with no known parent package
```

could you help me understand why? seems the relative import does not work in this way.

Reply:
I guess we're running into https://bugs.python.org/issue38457 here. Interesting that mgr/ is not affected.

Reply:
For py3 compat I prefer to use module full path over relative, it additionally avoid confusion for determining right namespace